src/CppParser/Bindings/CSharp/i686-pc-win32-msvc/Std.cs

```diff
@@ -8,6 +8,8 @@
 using System.Runtime.InteropServices;
 using System.Security;
 using System.Runtime.CompilerServices;
+using __CallingConvention = System.Runtime.InteropServices.CallingConvention;
```

Comment:
Please add global::.

Reply:
Why is that needed tho?

Reply:
To avoid conflicts with generated name-spaces and yes.

Reply:
Done. For some reason I thought that I could dodge using the global namespace if the statement was at the top of the file :smile: